src/App.jsx

```diff
@@ -1,33 +1,63 @@
+import { useState } from 'react';
+
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App = () => (
-  <div className="page">
-    <div className="page-content">
-      <div className="box">
-        <div className="field">
-          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-          <label htmlFor="search-query" className="label">
-            Search movie
-          </label>
-
-          <div className="control">
-            <input
-              type="text"
-              id="search-query"
-              className="input"
-              placeholder="Type search word"
-            />
+function filterMovie({ title, description }, query) {
+  const queryLower = query.toLowerCase();
+  const titleLower = title.toLowerCase();
+  const descriptionLower = description.toLowerCase();
+
+  return titleLower.includes(queryLower)
+    || descriptionLower.includes(queryLower);
+}
+
+function getPrepareMovies(movies, query) {
+  let preparedMovies = [...movies];
+
+  if (query) {
+    preparedMovies = preparedMovies
+      .filter(movie => filterMovie(movie, query));
+  }
+
+  return preparedMovies;
+}
```

Comment:
This code looks good, but if you invert the check you will make it better

Do you really need the `filterMovie` function? 
You are not going to reuse it and there is no hard logic, I think you can easily write it in the `.filter` callback.

```suggestion
function getFilteredMovies(movies, query) {
  if (!query) {
    return movies;
  }
  return movies.filter(movie => filterMovie(movie, query));
}
```